src/main/java/com/github/princesslana/smalld/GatewayIntent.java

```diff
@@ -24,12 +24,12 @@
   DIRECT_MESSAGE_TYPING(1 << 14);
 
   public static final int ALL = GatewayIntent.toMask(GatewayIntent.values());
-  public static final int PRIVILEGED = GatewayIntent.toMask(GUILD_PRESENCES, GUILD_MEMEBERS);
+  public static final int PRIVILEGED = GatewayIntent.toMask(GUILD_PRESENCES, GUILD_MEMBERS);
   public static final int UNPRIVILEGED = ALL ^ PRIVILEGED;
 
   private final int mask;
 
-  private GatewayIntent(int mask) {
+  GatewayIntent(int mask) {
```

Comment:
Why remote the private?

Reply:
I was modifying something before committing and just forgot to change it like this was in the original. Still I think there is no difference, because this constructor access is private by default.